src/CheckoutManager/V7/CommitOrderProcessor.php

```diff
@@ -329,7 +330,6 @@ public function cleanUpPendingOrders(): void
 
         //Abort orders with payment pending
         $list = $orderManager->buildOrderList();
-        $list->addFieldCollection('PaymentInfo');
```

Comment:
Why was this line removed?

Reply:
i removed this line because it serves no purpose.
The fieldCollection "PaymentInfo" is not used here and gets added in line 354 anyways (when it's needed).